lib/dependencies/AMDDefineDependency.js

```diff
@@ -57,7 +57,7 @@ AMDDefineDependency.Template = class AMDDefineDependencyTemplate {
 			],
 			lf: [
 				"var XXX;",
-				"!(XXX = #.call(exports, __webpack_require__, exports, module))"
+				"!(XXX = #.call(exports, __webpack_require__, exports, module)); XXX === undefined && (XXX = module.exports)"
```

Comment:
I must stay a single **expression**. Not a statement.

`define` could be used this way:

``` js
a && define(...) && b
```